Makefile

```diff
@@ -0,0 +1,13 @@
+build:
+	go build -o ./pctl cmd/pctl/main.go
+
+test: unit-test integration-test
+
+unit-test:
+	go test -count=1 ./pkg/...
```

Comment:
`-count=1` is the recommended way to disable test caching https://stackoverflow.com/questions/48882691/force-retesting-or-disable-test-caching

Reply:
I'm curious about _why_ we're doing this?

Are we seeing issues that require disabling test caching?

Reply:
this is very much just personal preferences, I have found it unreliable in the past. I think it was that I altered an artefact outside of the test directory, so it didn't re-run the tests as it didn't detect a difference.